.github/workflows/deploy.yml

```diff
@@ -90,6 +90,74 @@ jobs:
       - name: Run Deployment Script Previous Major Version
         if: ${{ github.ref != 'refs/heads/main' }}
         run: script/deploy-canary-demo -prev
+  deploy_ChimeSDKMeetings_ChimeSDKMediaPipelinesBeta_beta_demo:
```

Comment:
Would it be worth to have a pattern for all these names, something like:
For Beta JS SDK demos:
ChimeSDKMeetingsGamma_ChimeSDKMediaPipelinesGamma_beta_demo

For Gamma JS SDK demos:
ChimeSDKMeetings_ChimeSDKMediaPipelines_gamma_demo

For Prod JS SDK demos:
ChimeSDKMeetings_ChimeSDKMediaPipelines_prod_demo

And then match this to `name: ` field of the GitHub job as well as `env` variables if any. I think it would help this more easy to follow 😅 .

Currently, it is a bit confusing for example, the `env` variable says `ChimeSDKMeetings_ChimeSDKMediaPipelinesBeta` so what is the stage for `ChimeSDKMeetings` client, the environment variables tell it is gamma?

Let me know what do you think?

Reply:
I've updated the newly added stages, I'm hesitant to update the existing ones